bundles/org.openhab.binding.govee/src/main/java/org/openhab/binding/govee/internal/GoveeHandler.java

```diff
@@ -100,9 +101,7 @@ public class GoveeHandler extends BaseThingHandler {
     private final Runnable thingRefreshSender = () -> {
         try {
             triggerDeviceStatusRefresh();
-            if (!thing.getStatus().equals(ThingStatus.ONLINE)) {
-                updateStatus(ThingStatus.ONLINE);
-            }
+            updateStatus(ThingStatus.ONLINE);
```

Comment:
Btw, why was this changed? The same condition is in line 197, so the handler now has inconsistent logic for updating status to `ONLINE`.

Reply:
The framework already does this check in `ThingManagerImpl`, there is no need for this check. Bit minor though, without these checks in the bindings the code is cleaner. 
I also ran into a issue with the tests. I could fix it in the test, but prefer to remove this excess checking.

btw, i removed the other occurence too.